### DISCUSSION TITLE
Cloudflare help

### DISCUSSION THREAD

**Discussion by FranklinP44** [scosman/CMSaasStarter#95] [Q&A] [answered]
First, awesome template. As someone who’s building my first app the for the fourth time, this has been extremely helpful. I feel like I’m close (probably not) to finishing up the plumbing and am excited to move onto the UI stuff. I was able to get everything working on localhost, added Google as a login option, and even got it hosted on Cloudflare.

The issue I’m running into has something to do with the Sign In/Up buttons. On localhost, they work perfectly, but from the Cloudflare hosted site, I get this error as soon as I even hover over them.

`TypeError: Failed to resolve module specifier "@supabase/auth-ui-shared". Relative references must start with either "/", "./", or "../".`

I’m hoping someone might have some insights.

**Comment by scosman** [comment, marked as answer]
Diff it against master and see what's changed to look for clues. Master is what's deployed to https://saasstarter.work on CF so I know it  works.

If nothing stands out, double check you followed all the setup steps in the readme, and check the cloud flare deploy logs for clues.

**Reply by FranklinP44** [reply]
I wasn’t able to nail down exactly what I had done wrong, I got it working. Thanks for the reply!
